docs/getting-started/run-as-daemon.md

```diff
@@ -0,0 +1,26 @@
+# Run Dae as a Daemon Service
+
+[systemd](https://wiki.debian.org/systemd) allows you to create and manage services in extremely powerful and flexible ways.
+
+> **Note**
+> If your distribution's service manager is provided by systemd.
+
+Dae can be running as a daemon(systemd) service so that it can run at boot.
+
+## Setup
+
+```bash
+# download the sample systemd.service
+sudo curl -L -o /etc/systemd/system/dae.service https://github.com/v2rayA/dae/raw/main/install/dae.service
+
+# reload and restart daemon to take effect
+sudo systemctl daemon-reload
+sudo systemctl enable dae --now
+sudo systemctl status dae
+```
```

Comment:
Add some notes about config file (for examples: move the config file downloaded just now to `/etc/dae/`)

Reply:
Resolved in [eb8dc6c](https://github.com/v2rayA/dae/pull/25/commits/eb8dc6c4554d89391f4f7d20b9b4a0000a98126e)